### DISCUSSION TITLE
I'm not able to create an account in vercel

### DISCUSSION THREAD

**Discussion by night-slayer18** [vercel/community#3906] [Account Issue] [answered]
### Question

The vercel website shows account is blocked when i try to login whereas my colleague could do the same without any issues

**Comment by kgaurav152** [comment]
I too am facing the same issue, needed it in urgent, to deploy some project but not able to signup.

**Reply by night-slayer18** [reply]
I'm waiting for 2 days and no response I'm trying to host a next.js app but backed up due to this I don't know why they haven't fixed this as many people are experiencing this same issue.

**Comment by Marsuvees** [comment]
The vercel website shows account is blocked when i try to login or signup.

**Reply by night-slayer18** [reply]
I have been waiting for more than 2day now

**Comment by leerob** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Reply by night-slayer18** [reply]
Yeah i received a mail which said to wait and nothing else. I've read through the discussion and you guys haven't even responded to people who sent a mail 1 month ago. So I'm asking how long will it take and why is this issue not fixed in large scale.

**Reply by kgaurav152** [reply]
No, I haven't received any mail regarding this. Just saw the blocked msg on the signup page itself.

**Reply by night-slayer18** [reply]
You have to send a mail to them first after that a generated message will be sent as reply to you
